### DISCUSSION TITLE
D878UVII How can I load my own UserDB?

### DISCUSSION THREAD

**Discussion by swissalpsboy** [hmatuschek/qdmr#343] [General]
Hello Hannes

First of all, thanks for the great program!

I use the original Anytone CPS in VirtualBox to write the code plug. I create the data (including UserDB) itself with Calc and then import it into the CPS. This works quite well.

My problem is the UserDB. With my installation it takes about 6 hours until all entries are written. Your software does it much faster. Hi

Is it possible to upload a self created UserDB? If so, what is its structure?

Thanks and vy 73

**Comment by swissalpsboy** [comment]
Could have come up with this idea earlier, problem I solved as follows:

- Convert my .ods file to JSON
- replace the original user.json with my file
- start qdmr
- load user DB with standard function Device/Write call-sign DB into the radio.

done :-)

**Reply by hmatuschek** [reply]
I've created an issue for that (#345).
